source/Outfit.cpp

```diff
@@ -88,6 +87,31 @@ namespace {
 				thisMap.erase(it.first);
 		}
 	}
+	
+	// Summation that avoids a loss of trailing digits as possible.
```

Comment:
Shouldn't this be `// Summation that avoids loss of trailing digits as much as possible.`